assets/index.css

```diff
@@ -0,0 +1,1058 @@
+}
+
+.nw-ti,
+.magazine-title-inside,
+.editors-title-inside {
+  font-size: 1.188;
+  font-weight: bolder;
+  color: black;
+}
+
+.icon-newsletter {
+  width: 47px;
+  height: 28px;
+  background: url(https://g.newsweek.com/www/images/logo-n1.svg) center/contain no-repeat #f72210;
+}
+
+.sign-email {
+  width: 100%;
+  height: 32px;
+  color: #555;
+  font-size: 14px;
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: white;
+  border: 1px solid #e0e1e1;
+  text-indent: 9px;
+}
+
+.freesign-button {
+  width: 100%;
+  height: 32px;
+  color: white;
+  cursor: pointer;
+  font-weight: bold;
+  font-size: 0;
+  font-family: "Roboto Condensed", sans-serif;
+  background: url(https://g.newsweek.com/www/images/free-sign-up.svg) center/108px no-repeat #f61602;
+  border-radius: 3px;
+  border: none;
+}
+
+.title-magazine,
+.title-editors {
+  width: 100%;
+}
+
+.title-editors {
+  border-bottom: 1px solid #d9d9d9;
+}
+
+.cover-image-magazine,
+.image-magazine {
+  cursor: pointer;
+  width: 100%;
+  height: 130vw;
+  background-image: url("https://d.newsweek.com/en/full/1597761/cover-june-19-26-2020.jpg?w=360&h=480&q=90&f=4e759c49f52dd6986cbe78bcf012b463");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.image-magazine {
+  background-image: url("https://d.newsweek.com/en/full/1596483/china-hong-kong-future-opener.jpg?w=466&h=311&l=24&t=59&q=90&f=9b1b20405dea54feff9dcfaafbb625ac");
+  height: 60vw;
+}
+
+.im-mg1 {
+  background-image: url("https://d.newsweek.com/en/full/1595446/gwen-carr-george-floyd.jpg?w=466&h=311&q=90&f=855a80a46765783b52eb33cfe023908d");
+}
+
+.im-mg2 {
+  background-image: url("https://d.newsweek.com/en/full/1595075/minneapolis-riots-black-owned-business-george-floyd.jpg?w=466&h=311&q=90&f=8caca6e10b88e84a046a53aee9eb1317");
+}
+
+.im-mg3 {
+  background-image: url("https://d.newsweek.com/en/full/1596154/new-york-riots-protests-george-floyd.jpg?w=466&h=311&q=90&f=4eb4dc096ebfc1f7192093277b9e4cc1");
+}
+
+.im-mg4 {
+  background-image: url("https://d.newsweek.com/en/full/1597714/cul-williemays-banner.jpg?w=466&h=311&l=47&t=32&q=90&f=733c77c315444674a3122f9ef676959a");
+}
+
+.im-mg5 {
+  background-image: url("https://d.newsweek.com/en/full/1597051/cul-ps-apatow.jpg?w=466&h=311&l=46&t=41&q=90&f=b2214b8a1cab2bd6ae3ebdc7c832d7ba");
+}
+
+.im-mg6 {
+  background-image: url("https://d.newsweek.com/en/full/1597780/rocket-men.jpg?w=480&h=320&l=34&t=35&q=90&f=1c294eb910efd0b379abd65d84e159d0");
+}
+
+.all-features {
+  font-size: 1.2em;
+  font-weight: bolder;
+}
+
+.all-features:hover {
+  background-color: gray;
+  cursor: pointer;
+}
+
+.categories {
+  width: 100%;
+  font-size: 1.2em;
+  font-weight: bold;
+  font-family: 'Roboto Condensed', sans-serif;
+}
+
+.mag-tag {
+  background-color: #f72210;
+  width: fit-content;
+  height: fit-content;
+  display: flex;
+  justify-content: center;
+  color: white;
+  font-weight: bolder;
+  align-items: center;
```

Comment:
Bootstrap provides classes for `flexbox`. Remember to use them instead of adding them in your CSS file.

Reply:
I need it to change the display property in different screen size, so It cant be on the html

Reply:
You can use bootstrap classes to change the display properties on different sizes. For example if you want it to display flex on the large screen but not medium you can use `d-lg-flex`. Please see this [cheatsheet](https://hackerthemes.com/bootstrap-cheatsheet/) for more information.